integrations/cloudbuild/smoke-test.yaml

```diff
@@ -1,5 +1,5 @@
 steps:
-    - name: "connectedhomeip/chip-build-vscode:0.5.84"
+    - name: "connectedhomeip/chip-build-vscode:0.5.88"
```

Comment:
This will not work on 88 because it seems 88 will be unable to compile k32w.

I created #21112 to fix copy and paths, but it may still need a k32w change like #20471

Reply:
@andy31415
I changed all images to 0.5.89.
Lets wait till 0.5.89 build.